game_test.py

```diff
@@ -0,0 +1,446 @@
+import unittest
```

Comment:
you might also want to explore pytest

Reply:
Interesting, I will try that one in the next project!